.github/workflows/release.yaml

```diff
@@ -56,10 +56,19 @@ jobs:
           tag="$(cargo metadata --format-version 1 --no-deps | jq -r '.packages[0].version')"; \
           version="${tag}-snapshot"; \
         fi;
+        if [[ "$version" =~ '-' ]]; then \
```

Comment:
I don't understand why this check tells us that it's a pre-release.

Reply:
Because we follow server and pre-releases always follow Major.Minor.Patch-(pre-release tag) so something with a dash is a pre-release and something without isn't. Or am I missing something?

Reply:
Thanks, sounds good. I think I was the one missing something 😂

Reply:
Can we have that comment in code? It is not clear now, it won't be in the future.